app/models/course/group_user.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 class Course::GroupUser < ApplicationRecord
   after_initialize :set_defaults, if: :new_record?
-
-  enum :role, { normal: 0, manager: 1 }
+  acts_as_paranoid
+  enum role: { normal: 0, manager: 1 }
```

Comment:
Why are you reverting a change from rails 7.2 upgrade PR?

https://github.com/Coursemology/coursemology2/blame/8e2a6b9cc8c402da6884c212f044ea04d407b314/app/models/course/group_user.rb#L5

Reply:
Lemme revert it.